examples/seq2seq/finetune_trainer.py

```diff
@@ -106,6 +107,46 @@ class DataTrainingArguments:
     )
 
 
+def speed_metrics(mode, t0, n_objs):
```

Comment:
```suggestion
def speed_metrics(mode, start_time, num_samples):
```
I think we can have more readable names here :-) (If accepted some search and replace below is needed.)